python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -657,7 +657,7 @@ def getTableConstraints(self, table):
                   LEFT JOIN pg_class t ON c.conrelid = t.oid
                         LEFT JOIN pg_class t2 ON c.confrelid = t2.oid
                         JOIN pg_namespace nsp ON t.relnamespace = nsp.oid
-                        WHERE t.relname = %s %s """ % (con_col_name, self.quoteString(tablename), schema_where)
+                        WHERE c.contype <> 't' AND t.relname = %s %s """ % (con_col_name, self.quoteString(tablename), schema_where)
```

Comment:
Can you leave a comment that describes what `'t'` refers to, I think it's not obvious?

Reply:
Hi, @m-kuhn , I added a comment describing what 't' means (commit 304a3d9 ). Is it ok? Ready to merge and to backport to 3.12 and 3.10? Thank you.